shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -218,9 +218,15 @@ protected ScanInterpreter getInterpreter(final CommandLine cl, final String tabl
     Class<? extends ScanInterpreter> clazz = null;
     try {
       if (cl.hasOption(interpreterOpt.getOpt())) {
+        shellState.getReader().println(
+            "WARNING: Scan Interpreter option is deprecated and will be removed in 2.1.\n");
```

Comment:
I think deprecation is reasonable, but I'm not too keen on promising removal in a non-major release. Shell commands seem like public API to me, and should probably be treated that way. '3.0' would be better.

Also, I'm not sure if this is still the case, but I remember when I did the initial work on the shell that I tried to make the STDOUT be the result of the command being run, and STDERR (or at least, the logger, which can be configured to use STDERR) for informational stuff from the shell that isn't directly the output of a command. In principle, this would make it possible for scripting/automation, but I'm not sure if the shell has retained that design principle since the last time I did significant work on it.

Reply:
That fixes the promise about a specific version, but I'm still concerned about it being printed to STDOUT, mixed with command output. What was the reason for avoiding the logger method to print the message?

Reply:
I can add a logger message as well.  I was thinking printing to STDOUT was more obvious.

Reply:
The console logger is typically configured to go to STDOUT anyway, but at least they have the option of configuring it differently, in case they wanted to automate/script the shell and parse the output of their commands.

Reply:
Apparently I am making comments in the future... I will wait until future me returns.